plone/app/portlets/portletcontext.py

```diff
@@ -86,7 +86,7 @@ def _getGroupIds(self):
         # ambiguity - the Plone member-data version returns ids.
 
         for group in groups:
-            if type(group) not in StringTypes:
+            if not isinstance(group, six.text_type):
```

Comment:
I think this should be `six.string_types` instead (`StringTypes` ~= `basestring` builtin type)

Reply:
I'd rather use (six.text_type, six.binary_type) so that the method does not care if it is text or bytes as long as it is not groups. My guess is that `getGroups` returns text in py3.